### PR TITLE
Check tables in DB before running

### PR DESCRIPTION
**Pull request by asteel-gsa**

### CODE REVIEW

File: cmd/assets/db_tables.txt [hunk 1]
```diff
@@ -0,0 +1,134 @@
+census_duns21
```
Comment (asteel-gsa) [inline cmd/assets/db_tables.txt:1]: Ask Sudha if we can remove these tables entirely census_abc#

Comment (jadudm) [reply cmd/assets/db_tables.txt:1]: TBD, but not a blocker for review.

**Review by jadudm — DISMISSED** [review]
LGTM. Reviewed with asteel, based on changes from prior review w/  tballard.

**Review by jadudm — APPROVED** [review]
Fixed merge conflicts.

**Pull request merged**